rts/Lua/LuaVAO.cpp

```diff
@@ -74,6 +73,15 @@ LuaVAOs::~LuaVAOs()
 	luaVAOs.clear();
 }
 
+
+/***
+ *
+ * @function gl.GetVAO
+ * @treturn nil|VAO.VAO the VAO ref on success, else nil
```

Comment:
Maybe replace `VAO.VAO` and `VBO.VBO` with just `VAO` and `VBO`?

Reply:
Unfortunately won't be possible until I fork and rewrite LDoc, it uses lua modules as the grouping convention. So this references the `VAO` class in the `VAO` module.

Reply:
Ok, I was able to find a way with `@classmod`